### DISCUSSION TITLE
How to copy account address to Invoice on Yetiforce 6.1 ?

### DISCUSSION THREAD

**Discussion by apcloic** [YetiForceCompany/YetiForceCRM#13943] [How to] [answered]
Hi,

I don't know the reason but it looks like the buttons to copy address from contact & account has been removed in YF 6.1

So, does anybody knows what is the new way to copy client address to invoice now ?

Regards,

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
should work the same

**Comment by apcloic** [comment]
@mariuszkrzaczkowski 

Maybe I've missed something but it looks like in YF 6.1, you can only copy address between invoice shipping and invoice billing addresses while in YF 5.3 there was the option to copy address from contact/account address.

![image](https://user-images.githubusercontent.com/2930560/102695372-206b6900-4227-11eb-8b9f-1daa9c48317e.png)

**Reply by mariuszkrzaczkowski** [reply]
fixed in https://github.com/YetiForceCompany/YetiForceCRM/commit/e533fbd5b768f68f941e4838636c79ec84d7ab93

**Reply by apcloic** [reply]
@mariuszkrzaczkowski Perfect, ty !

PS : I don't know why but there is no "mark as answered" button on your last reply so I mark the previous one.
